### DISCUSSION TITLE
Nice

### DISCUSSION THREAD

**Discussion by wc-yaa** [blitz3d-ng/package#78] [General]
Hello, Blitz3D "NG" is really nice. The link was added to this webpage( https://itch.io/jam/jam-for-all-basic-dialects-6 ). Other BASIC hobbyists can find this nice programming language as well. Yes!

**Comment by kfprimm** [comment]
Ah, cool. I might participate in the jam.

**Reply by wc-yaa** [reply]
Hi, welcome. Developing entry for submitting to game jam will increase the exposure of Blitz3D "NG" effectively. It is because many indie game developers like to visit Itch. Blitz3D "NG" will be more well-known. Sure.
